draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -621,8 +621,16 @@ Other specifications or profiles of this specification may define alternative fo
 
 Security considerations in this section help achieve the following properties:
 
-* Claim values issued by the Issuer cannot be obtained by the adversary, if the Holder did not intend to disclose them.
-* Claim values that are not issued by the Issuer cannot be successfully verified by the Verifier, if the Holder forged the Disclosures.
+**Selective Disclosure:** An adversary in the role of the Verifier cannot obtain
+information from an SD-JWT about any claim name or claim value that was not
+explicitly disclosed by the Holder unless that information can be derived from
+other disclosed claims or external sources.
+
+**Integrity:** The claims verified by a Verifier are a subset of the claims
+released by the Issuer, even if the Holder is malicious (i.e., a Holder cannot
+modify claim names or claim values without the Verifier noticing during verification).
```

Comment:
I'm finding the main sentence kinda hard to process. What about just something inspired by the text you had in the parenthetical? 

```suggestion
**Integrity:** A malicious Holder cannot modify names or values of selectively disclosable claims without detection by the Verifier.
```

Reply:
I like Brian's suggested text. Would also suggest removing **Integrity**. Per a comment above, I find two introduced abstractions confusing.